src/Uno.UI/UI/Xaml/ApplicationActivity.Android.cs

```diff
@@ -96,19 +97,18 @@ public void ExitFullscreen()
 			Window.ClearFlags(WindowManagerFlags.Fullscreen);
 		}
 
-		private void OnLayoutChanged(Rect occludedRect)
+		private void OnLayoutChanged(Rect statusBar, Rect keyboard, Rect navigationBar)
```

Comment:
Why do we need to pass statusBar and navigationBar rects if we don't use them ?

Reply:
It is so that you can listen to the changes, in case, you do need it.